src/libs/TransactionUtils.js

```diff
@@ -142,6 +142,9 @@ function getUpdatedTransaction(transaction, transactionChanges, isFromExpenseRep
         updatedTransaction.receipt.state = CONST.IOU.RECEIPT_STATE.OPEN;
     }
 
+    // Always copy over the category for now until we have a way to edit it (Will be implemented in https://github.com/Expensify/App/issues/24464)
+    updatedTransaction.category = transaction.category;
```

Comment:
Is this... required, we clone the original transaction at the start of this method so that would be included.

What we have to do is to add it as a param to the edit money request call and create a new getter for it and return it in the getTransactionDetails https://github.com/Expensify/App/blob/67df8e9ff334899f836080dbd7e65dc6ffc137c1/src/libs/actions/IOU.js#L1058-L1071